src/components/RestaurantList.jsx

```diff
@@ -0,0 +1,82 @@
+import styles from '../css/RestaurantList.module.css';
+
+function RestaurantList() {
+    const restaurantList = getRestaurantList();
+
+    return (
+        <section className={styles['restaurant-list-container']}>
+            <ul className={styles['restaurant-list']}>
+                {restaurantList.map((restaurant, index) => (
+                    <RestaurantItem 
+                        key={index}
+                        name={restaurant.name}
+                        description={restaurant.description}
+                        category={restaurant.category}
+                        alt={restaurant.alt}
+                    />
+                ))}
+            </ul>   
+        </section>
+    );
+}
+
+function RestaurantItem({name, description, category, alt}) {
+    return (
+        <li className={styles['restaurant']}>
+          <div className={styles['restaurant__category']}>
+            <img src={`../../templates/category-${category}.png`} alt={alt} className={styles["category-icon"]}></img>
+          </div>
+          <div className={styles['restaurant__info']}>
+            <h3 className={`${styles.restaurant__name} text-subtitle`}>{name}</h3>
+            <p className={`${styles.restaurant__description} text-body`}>{description}</p>
+          </div>
+        </li>
+    );
+}
+
+function getRestaurantList() {
+    return [
+        {
+          name: "피양콩할마니",
+          description: `평양 출신의 할머니가 수십 년간 운영해온 비지 전문점 피양콩 할마니. 두부를 빼지 않은 되비지를 맛볼 수 있는 곳으로,
+          ‘피양’은 평안도 사투리로 ‘평양’을 의미한다. 딸과 함께 운영하는 이곳에선 맷돌로 직접 간 콩만을 사용하며, 일체의 조미료를 넣지 않은 건강식을 선보인다.
+          콩비지와 피양 만두가 이곳의 대표 메뉴지만, 할머니가 옛날 방식을 고수하며 만들어내는 비지전골 또한 이 집의 역사를 느낄 수 있는 특별한 메뉴다.
+          반찬은 손님들이 먹고 싶은 만큼 덜어 먹을 수 있게 준비돼 있다.`,
+          category: "korean",
+          alt: "한식"
+        },
+        {
+          name: "친친",
+          description: `Since 2004 편리한 교통과 주차, 그리고 관록만큼 깊은 맛과 정성으로 정통 중식의 세계를 펼쳐갑니다.`,
+          category: "chinese",
+          alt: "중식"
+        },
+        {
+          name: "잇쇼우",
+          description: `잇쇼우는 정통 자가제면 사누끼 우동이 대표메뉴입니다. 기술은 정성을 이길 수 없다는 신념으로 모든 음식에 최선을 다하는 잇쇼우는
+          고객 한분 한분께 최선을 다하겠습니다.`,
+          category: "japanese",
+          alt: "일식"
+        },
+        {
+          name: "이태리키친",
+          description: `늘 변화를 추구하는 이태리키친입니다.`,
+          category: "western",
+          alt: "양식"
+        },
+        {
+          name: "호아빈 삼성점",
+          description: `푸짐한 양에 국물이 일품인 쌀국수.`,
+          category: "asian",
+          alt: "아시안"
+        },
+        {
+          name: "도스타코스 선릉점",
+          description: `멕시칸 캐주얼 그릴.`,
+          category: "etc",
+          alt: "기타"
+        }
+      ];
+}
+
```

Comment:
레스토랑이 늘어날 수록 데이터가 커질 수 있어서 저도 분리를 해야할까 싶네요..!!

Reply:
레스토랑 정보를 함수로 만드셨군요? 저는 다른 방식으로 했답니다. js의 모듈화 기능을 이용했어요. 

```javascript
//data.js
export const restaurants = [
        {
            key: "r1",
            name: "피양콩할마니",
            description: `평양 출신의 할머니가 수십 년간 운영해온 비지 전문점 피양콩 할마니. 두부를 빼지 않은 되비지를 맛볼 수 있는 곳으로,
            ‘피양’은 평안도 사투리로 ‘평양’을 의미한다. 딸과 함께 운영하는 이곳에선 맷돌로 직접 간 콩만을 사용하며, 일체의 조미료를 넣지 않은 건강식을 선보인다.
            콩비지와 피양 만두가 이곳의 대표 메뉴지만, 할머니가 옛날 방식을 고수하며 만들어내는 비지전골 또한 이 집의 역사를 느낄 수 있는 특별한 메뉴다.
            반찬은 손님들이 먹고 싶은 만큼 덜어 먹을 수 있게 준비돼 있다.`,
            category: "korean",
            alt: "한식"
        },
        {
            key: "r2",
            name: "친친",
            description: `Since 2004 편리한 교통과 주차, 그리고 관록만큼 깊은 맛과 정성으로 정통 중식의 세계를 펼쳐갑니다.`,
            category: "chinese",
            alt: "중식"
        },
        {
            key: "r3",
            name: "잇쇼우",
            description: `잇쇼우는 정통 자가제면 사누끼 우동이 대표메뉴입니다. 기술은 정성을 이길 수 없다는 신념으로 모든 음식에 최선을 다하는 잇쇼우는
            고객 한분 한분께 최선을 다하겠습니다.`,
            category: "japanese",
            alt: "일식"
        },
        {
            key: "r4",
            name: "이태리키친",
            description: `늘 변화를 추구하는 이태리키친입니다.`,
            category: "western",
            alt: "양식"
        },
        {
            key: "r5",
            name: "호아빈 삼성점",
            description: `푸짐한 양에 국물이 일품인 쌀국수.`,
            category: "asian",
            alt: "아시안"
        },
        {
            key: "r6",
            name: "도스타코스 선릉점",
            description: `멕시칸 캐주얼 그릴.`,
            category: "etc",
            alt: "기타"
        }
      ];
}
```

이렇게 data.js 파일을 따로 두고 import/export 해가며 사용했어요. 이렇게 하는 방식이 코드 분리와 코드가독성 측면에서 더 나을 거라고 생각하는데 준수님 생각은 어떤지 듣고 싶습니다!

Reply:
저도 이 방식으로 진행하는 방향이 좋은거 같아요!!
카테고리를 이 방식을 적용해봤는데, 레스토랑도 이 방식으로 진행해야겠어요!😊